src/modules/dashboard/components/ColonyActions/ColonyActions.tsx

```diff
@@ -173,21 +174,37 @@ const ColonyActions = ({
   ]);
 
   /* Needs to be tested when all action types are wirde up & reflected in the list */
-  const filteredActions = useMemo(
-    () =>
-      !ethDomainId
-        ? actions
-        : actions.filter(
-            (action) =>
-              Number(action.fromDomain) === ethDomainId ||
-              /* when no specific domain in the action it is displayed in Root */
-              (ethDomainId === 1 && action.fromDomain === undefined) ||
-              /* when transfering funds the list shows both sender & recipient */
-              (action.actionType === ColonyActionTypes.MoveFunds &&
-                Number(action.toDomain) === ethDomainId),
-          ),
-    [ethDomainId, actions],
-  );
+  const filteredActions = useMemo(() => {
+    let filterActions = actions;
```

Comment:
I would avoid let whenever possible, just do const filterActions = [and condition here] with ternary operator